app/services/invitation_service.rb

```diff
@@ -32,6 +32,14 @@ def self.invite_to_group(recipient_emails: nil,
     end
   end
 
+  def self.resend(invitation)
```

Comment:
I think this should be `resend(invitation: )` maybe? so it's in line with the other methods?